api/src/__tests__/widgetMutations.test.ts

```diff
@@ -1332,6 +1378,132 @@ describe('lead', () => {
       associatedFieldId: customProperty._id
     });
 
+    const companyNameField = await fieldFactory({
+      type: 'companyName',
+      contentTypeId: form._id,
+      validation: 'text',
```

Comment:
Too many duplicated params. Consider the following:
const params = { validation: 'text', isRequired: true, contentTypeId: form._id }

const field = await fieldFactory({
    ...params,
   // other fields
})